docs/2-virtual-machines-containers/2.5.2-kubernetes.md

```diff
@@ -27,11 +27,11 @@ The **nodes** in a cluster are the machines (VMs, physical servers, etc) that ru
 ![](img2/delivery-k8s-node_dark.svg ':class=dark-mode-img-center')
 
 # Design
-Kubernetes is declarative by design. When you want to change something in the cluster, you simply create/update/delete the resources from etcd via the API server. Various controllers then respond to this change by converging the system to comply with the newly declared state. To best understand, check out this drawing from [Julia Evans](https://jvns.ca/blog/2017/06/04/learning-about-kubernetes/):
+Kubernetes is declarative by design. When you want to change something in the cluster, you simply create/update/delete the resources from etcd via the API server. Various controllers then respond to this change by converting the system to comply with the newly declared state. To best understand, check out this drawing from [Julia Evans](https://jvns.ca/blog/2017/06/04/learning-about-kubernetes/):
```

Comment:
I think converging works better here

Reply:
Sounds good. How do I approve the request?